perf/benchmark/runner/runner.py

```diff
@@ -309,6 +311,15 @@ def generate_nighthawk_cmd(self, cpus, conn, qps, jitter_uniform, duration, labe
 
         return nighthawk_cmd
 
+    def generate_policies(self, security_cmd):
+        os.system("go build {path}generate_policies.go {path}generate.go".format(path=LOCAL_GENERATE_POLICIES))
```

Comment:
I updated the code to generate the binary inside of the generate_policies folder instead of the runner folder. We could compile the binary in another temporary directory

Reply:
use `go run` instead?

Reply:
Go run would be best, will update accordingly

Reply:
Updated to `go run`